### PR TITLE
add FeaturedActivity component

### PR DESCRIPTION
**Pull request by JersinFong**
Hi, please check my first pull request
a new component for issue #16

### CODE REVIEW

**Comment by JersinFong** [comment]
image
delt with issue#19 too

File: src/components/shared/high/AvatarGroup.js [hunk 1]
```diff
@@ -38,6 +50,11 @@ const AvatarGroup = ({
 			<Stack isInline {...props}>
 				{clones}
 			</Stack>
+			{showNames && <Names>{names.map((m, i)=>{
```
Comment (poroia) [inline src/components/shared/high/AvatarGroup.js:53]: In the future, there may potentially be many, many avatars, so the `max` property was implemented to account for a huge number of names. We should probably limit the max number of names with `max`, too.

File: src/components/shared/high/FeaturedActivity.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+const Pic = styled.div`
+    width:100%;
+    height: 8.5em;
+    background-size: 100% auto;
+    background-image: url(${props => props.imgURL});
+    background-position: center;
+    border-radius: 0.5em 0.5em 0em 0em;
+`
+const ContentCard = styled.div`
+    width:100%
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:24]: Missing semicolon

File: src/components/shared/high/FeaturedActivity.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    padding: 2em 2em;
+    background: #FFFFFF;
+    border-radius: 0.5em;
+`
+
+const CardTtile = styled.p`
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:30]: Misspelling (also when used, too)

File: src/components/shared/high/FeaturedActivity.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    line-height: 41px
+    color: #000000;
+    margin: 0px;
+`
+
+const CardDescription = styled.p`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:44]: We should use use `percents`/`ems` here for the `font-size` (also for the line-height) to follow our responsive model

File: src/components/shared/high/FeaturedActivity.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+`
+const InlineDiv = styled.div`
+    margin: 1em auto;
+    display: flex;
+`
+const CommitText = styled.p`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 10px;
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:57]: We should use use percents/ems here for the font-size (also for the line-height) to follow our responsive model

File: src/components/shared/high/FeaturedActivity.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    line-height: 14px;
+    color: #6D6D6D;
+`
+
+const DateText = styled.p`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 10px;
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:66]: We should use use percents/ems here for the font-size (also for the line-height) to follow our responsive model

File: src/components/shared/high/FeaturedActivity.js [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+const Wrapper = styled.div`
+    flex: 60%;
+    display: flex;
+`
+
+const FeaturedActivity = ({
+    imgURL,
+    titleText,
+    contentText,
+    owners = [],
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:84]: Sorry, it wasn't explicitly mentioned, but if we can change `owners` to `authors`, that would be great!

File: src/components/shared/high/FeaturedActivity.js [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+            <ContentCard>
+                <CardTtile>{titleText}</CardTtile>
+                <InlineDiv>
+                    <AvatarWrappr>
+                        <AvatarGroup size="1.1em" spacing='-0.5em' names={owners} showNames={true}>
+                            {owners.map(owner => <Avatar key={owner} name={owner} />)}
+                        </AvatarGroup>
+                    </AvatarWrappr>
+                    <Wrapper>
+                        <IconLine icon = "⚡" gap = '0.2em'></IconLine>
```
Comment (poroia) [inline src/components/shared/high/FeaturedActivity.js:99]: `IconLine` is only needed when you need to stick a component as the `icon` inline with text. As @lazyplatypus mentioned, we should use an svg icon to maintain continuity on devices. For now, you can directly pull an svg from online and put it into the `src/assets/icons` folder and import it and put it into an `Icon` component to put in as the `icon` prop for the `IconLine` component.

File: src/components/shared/high/AvatarGroup.js (hunk 1 above)
Comment (JersinFong) [reply src/components/shared/high/AvatarGroup.js:53]: image
Hi, thanks for the suggestion, I handled this inside map function: if current index > max, map will return null, which, by default of JSX, will not show on the screen. Is this way okay or we'd better to cut the array to fixed size(max)

**Pull request merged**